logs/debug.go

```diff
@@ -35,15 +35,13 @@ func PrintDebugInfo(logFileContents string, logLines []state.LogLine, samples []
 	if len(unclassifiedLogLines) > 0 {
 		fmt.Printf("\nUnclassified log lines:\n")
 		for _, logLine := range unclassifiedLogLines {
-			fmt.Printf("%s\n", logFileContents[logLine.ByteStart:logLine.ByteEnd])
-			fmt.Printf("  Level: %s\n", logLine.LogLevel)
-			fmt.Printf("  Content: %#v\n", logFileContents[logLine.ByteContentStart:logLine.ByteEnd])
+			fmt.Printf("  %s: %s", logLine.LogLevel, logLine.Content)
```

Comment:
I haven't tried running the debug command yet. Someone might have a preference on the output format.

One downside to this PR is we can no longer debug the entire log content including the unparsed log prefix.

Reply:
I think that should be fine - the main use case for this is when the logs can't be analyzed, and that should still work with this updated output.